index.html

```diff
@@ -216,6 +216,55 @@ <h2>Skills</h2>
                     </div>
                 </div>
             </section>
+            <!------------------------------------Contact section------------------------------------>
+            <section class="contact-form">
+                <div class="contact-header">
+                    <h1>I'm always interested in hearing about new projects, so if you'd 
+                    like to chat please get in touch</h1>
```

Comment:
- [ ] I am excited that you considered the usage of `hx` tag for the headings. However, I suggest you consider using other `hx` tags instead of `h1` tags.  There should be only one `h1` tag on every page. This is because the screen readers target the `h1` tag which is the main heading and primarily found in the banner, to give people with poor sight, an overview of what the page is the website is all about ☺️.